tools/sync.js

```diff
@@ -9,63 +9,88 @@ var etherUnits = require("../lib/etherUnits.js");
 var BigNumber = require('bignumber.js');
 var _ = require('lodash');
 
-var async = require('async');
+var asyncL = require('async');
 var Web3 = require('web3');
 
 var mongoose        = require( 'mongoose' );
 var Block           = mongoose.model( 'Block' );
 var Transaction     = mongoose.model( 'Transaction' );
 var Account         = mongoose.model( 'Account' );
 
+const normalizeTX = async (txData, blockData) => {
```

Comment:
this function seems to be defined the same in 2 different files, should we abstract it out?

Reply:
I think so :)